generator/directory.ts

```diff
@@ -90,9 +90,13 @@ const colors = [
     "light ${colour}"
 ];
 
-const numbersFromOneToFive = [...Array(5).keys()].map(x => (x + 1).toString());
+const numbersFromOneToFive = ["1", "2", "3", "4", "5"];
```

Comment:
But but but what if we wanted `numbersFromOneToFive` to actually include numbers 1..N this is totally inflexible! /s

Reply:
Oh yeah this didn't play well with typescript for some reason? Idk what the problem was